web/frontend/src/components/arm/index.svelte

```diff
@@ -88,14 +87,6 @@ let modifyAllStatus: ArmStatus = {
 
 const armClient = new ArmClient($robotClient, name, { requestLogger: rcLogConditionally });
```

Comment:
(Question applies everywhere)

If `$robotClient` is `null` until the robot connects, why isn't this client created reactively? Would making the `arcClient` reactive give us a path to solve the "card talks to the robot too early" problem?

Reply:
`$robotClient` as a store is actually a bit misleading, it's declared immediately in `robot-client.svelte`, so it should never be null.